src/Icons/README.md

```diff
@@ -0,0 +1,103 @@
+# Symfony UX Icons
+
+## Installation
+
+```bash
+composer require symfony/ux-icons
+```
+
+## Icons?
+
+No icons are provided by this package but there are several ways to include and render icons. 
+
+### Local SVG Icons
+
+Add your svg icons to the `assets/icons/` directory and commit them.
+The name of the file is used as the _name_ of the icon (`name.svg` will be named `name`).
+If located in a subdirectory, the _name_ will be `sub-dir:name`.
+
+### Iconify Icons
+
+[Iconify Design](https://iconify.design/) is a huge searchable repository of icons from many different icon sets.
+This package provides a way to include any icon found on this site _on-demand_.
+
+1. Visit [Iconify Design](https://icon-sets.iconify.design/) and search for an icon
+   you'd like to use. Once you find one you like, visit the icon's profile page and use the widget
+   to copy its name. For instance, https://icon-sets.iconify.design/flowbite/user-solid/ has the name
+   `flowbite:user-solid`.
+2. Just use this name in the [`ux_icon()`](#usage) function and the icon will be fetched (and cached)
+   from the Iconify API.
+3. That's it!
+
+> [!NOTE]
+> [Local SVG Icons](#local-svg-icons) of the same name will have precedence over _Iconify-on-demand_ icons.
+
+#### Import Command
+
+You can import any icon from Iconify to your local directory using the `ux:icons:import` command:
+
+ ```bash
+ bin/console ux:icons:import flowbite:user-solid # saved as `flowbite/user-solid.svg` and name is `flowbite:user-solid`
+
+ # import several at a time
+ bin/console ux:icons:import flowbite:user-solid flowbite:home-solid
+ ```
+
+## Usage
```

Comment:
Kevin, thanks for this contribution. I like this a lot.

I have a question about how this would work in practice. When the docs say _"it renders the XXX icon"_ does it mean:

* It embeds the `<svg ...> ... </svg>` contents where you called the `ux_icon()` function
* It makes an Ajax request to fetch the contents and embeds it?
* It just links to some SVG asset URL?

I think it'd be more clear to me if you showed the full real output of these functions. This would also be useful to know where the CSS classes are applied, etc. Thanks.

Reply:
Thanks Javier, indeed it renders the svg tag in your html. I'll update to make that clear.

Reply:
I've updated the readme to explain with an example at the very top - let me know if you think this is sufficient.

Reply:
Yes, it's very clear now. Thanks!